projects/framework-nuke/source/ftrack_framework_nuke/__init__.py

```diff
@@ -176,27 +200,22 @@ def bootstrap_integration(framework_extensions_path):
             else:
                 ftrack_menu.addCommand(
                     tool['label'],
-                    f'{__name__}.onRunToolCallback("{name}","{dialog_name}", {options})',
+                    f'{__name__}.onRunToolCallback("{name}","{label}",True,"{action}","{dialog_name}", {options})',
                 )
+        startup_tools.append(
+            [
+                name,
+                label,
+                run_on == "startup",
+                action,
+                dialog_name,
+                options,
+            ]
+        )
```

Comment:
Will improve this with the separated actions, and run_tool. But that will happen on the base branch.